Marlin/src/lcd/e3v2/common/encoder.cpp

```diff
@@ -85,7 +85,7 @@ EncoderState encoderReceiveAnalyze() {
       next_click_update_ms = millis() + 300;
       Encoder_tick();
       #if PIN_EXISTS(LCD_LED)
-        //LED_Action();
+        LED_Action();
```

Comment:
Have you tested this to confirm it behaves as expected? If yes, what does it actually do?

Reply:
unable to actually test